scripts/build.ts

```diff
@@ -8,21 +8,27 @@ const __dirname = path.dirname(url.fileURLToPath(import.meta.url));
 
 const dist = path.join(__dirname, "..", "dist");
 
+const watch = process.argv.includes("--watch");
```

Comment:
Could you add a `build:watch` script to `package.json` so it's "documented" that there's a watch flag?

Reply:
done